CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Revision history for nothunks
 
-## next version
+## 0.2.0 -- 2024-01-27
```

Comment:
Add next version section

Reply:
I don't want to when publishing on `hackage`.

Reply:
That makes sense 🙂